### ISSUE TITLE
Block -> Entity Mutations

### ISSUE THREAD

**Issue by TemperedFool** [Team-Resourceful/ResourcefulBees#212]
**Is your feature request related to a problem? Please describe.**
I'm working on Bee Mutations for Enigmatica 6, and was hoping to be able to convert blocks to entities. Not a "problem" per se - I'll do block -> spawn egg for now - but converting directly to the entity would be more elegant.

**Describe the solution you'd like**
Allow me to set up a "ENTITY" type mutation that can take a block (or block tag) as input.

**Describe alternatives you've considered**
I have a functional alternative - Block -> Spawn Egg. As I said, it's just not as clever.

**Additional context**
Thank you for considering this at all! Clearly not a mission-critical addition, but I'll very gladly convert everything over from spawn eggs to entities if you add this!

**Comment by Epic428** [comment]
I’ll consider this. I’ve been a bit apprehensive to having entity mutations beyond what we have now Bc they’re a huge pain to work with.

**Comment by TemperedFool** [comment]
There is also the real risk of the entity suffocating, but I feel like that's a logistical challenge for the player.

If the challenge working with them is implementing and debugging them, I totally get it; otherwise so far working with them in game (after adding them) has been pretty easy IMO.

I'm interested in it as a fun way to acquire rare mobs (bees included), and them bursting out of the block is definitely cooler than a spawn egg dropping 😉

**Comment by Epic428** [comment]
Yeah it’s more so the issue with implementation/debugging granted most of the code already exists for both sides of the coin it’s more so just refactoring the bits into a new recipe for jei and adding in the extra logic on the entity. I agree though that it would be useful for a dev to have the option.

side note: entity mutations to a dragon are quite fun.

**Comment by TemperedFool** [comment]
Hmm, that is also quite clever. This may give me a new side project to general bee improvements...

**Comment by Epic428** [comment]
Yeah I took out a pretty large chunk of my test world mutating a cow to a dragon lmao. I’ll look at adding in the block -> entity mutations since it’s been requested before. I have other more pressing bugs to fix first though.

**Comment by TemperedFool** [comment]
"Eventually" works fine. I'd appreciate a ping if you end up with a dev version or something I can start testing recipes against. Tysm!

**Comment by Epic428** [comment]
No worries I’ll let you know when I have something for you to play with 🙂

**Comment by TemperedFool** [comment]
In thinking about this, item -> entity mutations would also be pretty cool, and would be less likely to suffer from suffocation...
